src/main/java/org/example/springproject/controller/BookController.java

```diff
@@ -33,12 +34,13 @@ public BookDto getBookById(@PathVariable Long id) {
     }
 
     @PostMapping()
```

Comment:
Also, you can add `@PathVariable @Positive Long id` where is needed and `@Validated` under the controller